paddle/scripts/travis/check_style.sh

```diff
@@ -11,18 +11,21 @@ set -e
 # install glide
 curl https://glide.sh/get | bash
 eval "$(GIMME_GO_VERSION=1.8.3 gimme)"
-go get -u github.com/alecthomas/gometalinter
+
+# set up go environment for running gometalinter
+mkdir -p $GOPATH/src/github.com/PaddlePaddle/
+ln -sf $TRAVIS_BUILD_DIR $GOPATH/src/github.com/PaddlePaddle/Paddle
+cd  $GOPATH/src/github.com/PaddlePaddle/Paddle/go; glide install; cd -
+
+go get github.com/alecthomas/gometalinter
 gometalinter --install
 
 cd $TRAVIS_BUILD_DIR
 export PATH=/usr/bin:$PATH
 pre-commit install
 clang-format --version
 
-# set up go environment for running gometalinter
-mkdir -p $GOPATH/src/github.com/PaddlePaddle/
-ln -sf $TRAVIS_BUILD_DIR $GOPATH/src/github.com/PaddlePaddle/Paddle
-cd  $GOPATH/src/github.com/PaddlePaddle/Paddle/go; glide install; cd -
+
```

Comment:
Please delete the redundant blank line.

Reply:
Will update when adding back `gometalinter`.